be/src/olap/tablet_schema.h

```diff
@@ -169,6 +172,7 @@ class TabletSchema {
     size_t _num_short_key_columns = 0;
     size_t _num_rows_per_row_block = 0;
     CompressKind _compress_kind = COMPRESS_NONE;
+    segment_v2::CompressionTypePB _compression_type = segment_v2::CompressionTypePB::NO_COMPRESSION;
```

Comment:
why defaut is NO_COMPRESSION

Reply:
I think it is just a zero value. It will be set after creation like _compress_kind.